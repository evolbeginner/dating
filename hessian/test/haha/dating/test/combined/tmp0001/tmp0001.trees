  1

(t1, t5, ((t4, t3), t2));

