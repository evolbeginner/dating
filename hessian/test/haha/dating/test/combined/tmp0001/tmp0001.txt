
         5          1  P

t1                              A
t5                              A
t4                              A
t3                              A
t2                              A


