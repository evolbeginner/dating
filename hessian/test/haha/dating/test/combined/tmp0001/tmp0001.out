AAML (in paml version 4.10.6, November 2022)  tmp0001.txt
Model: Empirical,  (/mnt/hd1/home/sishuo/software/phylo/paml/latest/dat/lg.dat) dGamma (ncatG=4) ns =   5  ls =   1


Frequencies..
                                    A      R      N      D      C      Q      E      G      H      I      L      K      M      F      P      S      T      W      Y      V
t1                             0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000
t5                             0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000
t4                             0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000
t3                             0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000
t2                             0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000

Homogeneity statistic: X2 = 0.00000 G = 0.00000 

Average                        0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000 0.05000

# constant sites:      0 (0.00%)
AA distances (raw proportions of different sites)

t1             
t5               0.0000
t4               0.0000  0.0000
t3               0.0000  0.0000  0.0000
t2               0.0000  0.0000  0.0000  0.0000

TREE #  1:  (1, 2, ((3, 4), 5));   MP score: 0
lnL(ntime:  7  np:  8):     -0.000000      +0.000000
   6..1     6..2     6..7     7..8     8..3     8..4     7..5  
 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.500000
